packages/meteor-accounts-saml/saml_server.js

```diff
@@ -182,61 +186,64 @@ middleware = function (req, res, next) {
 			return;
 		}
 
-		if (!samlObject.actionName)
-			throw new Error("Missing SAML action");
+		if (!samlObject.actionName) {
+			throw new Error('Missing SAML action');
+		}
 
-		console.log(Accounts.saml.settings.providers)
-		console.log(samlObject.serviceName)
+		console.log(Accounts.saml.settings.providers);
+		console.log(samlObject.serviceName);
 		var service = _.find(Accounts.saml.settings.providers, function (samlSetting) {
 			return samlSetting.provider === samlObject.serviceName;
 		});
 
 		// Skip everything if there's no service set by the saml middleware
-		if (!service)
-			throw new Error("Unexpected SAML service " + samlObject.serviceName);
+		if (!service) {
+			throw new Error('Unexpected SAML service ' + samlObject.serviceName);
+		}
+		var _saml;
 		switch (samlObject.actionName) {
-		case "metadata":
+		case 'metadata':
 			_saml = new SAML(service);
-			service.callbackUrl = Meteor.absoluteUrl("_saml/validate/" + service.provider);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["Meteor" is not defined.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2313371378/issues/source?bid=3199634&fileBranchId=3199636#l207)